content/references/protocol-reference/transactions/transaction-metadata.md

```diff
@@ -68,6 +68,15 @@ A `ModifiedNode` object contains the following fields:
 
 **Note:** If the modified ledger object has `PreviousTxnID` and `PreviousTxnLgrSeq` fields, the transaction always updates them with the transaction's own identifying hash and the index of the ledger version that included the transaction, but these fields' new value is not listed in the `FinalFields` of the `ModifiedNode` object, and their previous values are listed at the top level of the `ModifiedNode` object rather than in the nested `PreviousFields` object.
 
+## NFT Fields
+
+Transactions involving NFTs can contain the following fields in the metadata. These values are added by the Clio server at request time and are not stored in the hashed binary  metadata:
+
+| Field               | Value                     | Description                |
+|:--------------------|:--------------------------|:---------------------------|
+| `nftoken_id`        | String                    | Shows the `NFTokenID` for the `NFToken` that changed on the ledger as a result of the transaction. See [NFTokenID](nftoken.html#nftokenid) |
```

Comment:
I think we should be more specific here. This metadata field is only present if the tx is a `NFTokenMint` or `NFTokenAcceptOffer`, so that users know that they need to catch this field _only_ for these two transactions.